r_examples/r_sagemaker_binary_classification_algorithms/R_binary_classification_algorithms_comparison.ipynb

```diff
@@ -0,0 +1,2004 @@
+{
```

Comment:
<p>Preferred to use "you" or "we" instead of "I" throughout the notebook, since when the customer is running the notebook, they are also participating.</p><p> </p><p>For instance, "In the notebook tutorial, we build 3 classification models...", "The prediction we will perform", etc.</p><p> </p><p>Typo - <code>lenght</code> -&gt; <code>length</code> ﻿</p><p> </p><p>For consistency, use <code>versicolor</code>  rather than <code>versicolour</code>  <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2926/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>